[BUG] Installation Error on Unraid OS

## Bug summary

I used to be able to install rdiff-backup on Unraid OS just fine for ages.....within the past 6 months or so....that has all changed and it no longer installs successfully

## Version, Python, Operating System

```
Unraid OS
Python 3.9.0
```

## What happened and what did you expect?

```
 pip3 install rdiff-backup
Collecting rdiff-backup
  Using cached rdiff-backup-2.0.5.tar.gz (456 kB)
Requirement already satisfied: setuptools in /usr/lib64/python3.9/site-packages (from rdiff-backup) (50.3.2)
Building wheels for collected packages: rdiff-backup
  Building wheel for rdiff-backup (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /usr/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-czl32xcz/rdiff-backup/setup.py'"'"'; __file__='"'"'/tmp/pip-install-czl32xcz/rdiff-backup/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-9_2olrd2
       cwd: /tmp/pip-install-czl32xcz/rdiff-backup/
  Complete output (56 lines):
  running bdist_wheel
  running build
  running build_py
  running build_templates
  templating tools/rdiff-backup.spec.template -> build/rdiff-backup.spec
  creating build
  templating tools/rdiff-backup.spec.template-fedora -> build/rdiff-backup.fedora.spec
  templating docs/rdiff-backup.1 -> build/rdiff-backup.1
  templating docs/rdiff-backup-statistics.1 -> build/rdiff-backup-statistics.1
  creating build/lib.linux-x86_64-3.9
  creating build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/win_acls.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/increment.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/Security.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/log.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/connection.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/librsync.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/manage.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/eas_acls.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/TempFile.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/longname.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/Rdiff.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/Globals.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/robust.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/selection.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/restore.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/backup.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/rorpiter.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/iterfile.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/SetConnections.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/user_group.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/Time.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/regress.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/FilenameMapping.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/__init__.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/metadata.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/Main.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/Hardlink.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/fs_abilities.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/hash.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/statistics.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/rpath.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  copying src/rdiff_backup/compare.py -> build/lib.linux-x86_64-3.9/rdiff_backup
  running build_ext
  building 'rdiff_backup.C' extension
  creating build/temp.linux-x86_64-3.9
  creating build/temp.linux-x86_64-3.9/src
  gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/python3.9 -c src/cmodule.c -o build/temp.linux-x86_64-3.9/src/cmodule.o
  gcc -pthread -shared build/temp.linux-x86_64-3.9/src/cmodule.o -L/usr/lib64 -o build/lib.linux-x86_64-3.9/rdiff_backup/C.cpython-39-x86_64-linux-gnu.so
  building 'rdiff_backup._librsync' extension
  gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/python3.9 -c src/_librsyncmodule.c -o build/temp.linux-x86_64-3.9/src/_librsyncmodule.o
  src/_librsyncmodule.c:25:10: fatal error: librsync.h: No such file or directory
     25 | #include <librsync.h>
        |          ^~~~~~~~~~~~
  compilation terminated.
  error: command '/usr/bin/gcc' failed with exit code 1
  ----------------------------------------
  ERROR: Failed building wheel for rdiff-backup
  Running setup.py clean for rdiff-backup
Failed to build rdiff-backup
Installing collected packages: rdiff-backup
    Running setup.py install for rdiff-backup ... error
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-czl32xcz/rdiff-backup/setup.py'"'"'; __file__='"'"'/tmp/pip-install-czl32xcz/rdiff-backup/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-_znlo7sn/install-record.txt --single-version-externally-managed --compile --install-headers /usr/include/python3.9/rdiff-backup
         cwd: /tmp/pip-install-czl32xcz/rdiff-backup/
    Complete output (56 lines):
    running install
    running build
    running build_py
    running build_templates
    templating tools/rdiff-backup.spec.template -> build/rdiff-backup.spec
    creating build
    templating tools/rdiff-backup.spec.template-fedora -> build/rdiff-backup.fedora.spec
    templating docs/rdiff-backup.1 -> build/rdiff-backup.1
    templating docs/rdiff-backup-statistics.1 -> build/rdiff-backup-statistics.1
    creating build/lib.linux-x86_64-3.9
    creating build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/win_acls.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/increment.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/Security.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/log.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/connection.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/librsync.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/manage.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/eas_acls.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/TempFile.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/longname.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/Rdiff.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/Globals.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/robust.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/selection.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/restore.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/backup.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/rorpiter.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/iterfile.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/SetConnections.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/user_group.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/Time.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/regress.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/FilenameMapping.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/__init__.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/metadata.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/Main.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/Hardlink.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/fs_abilities.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/hash.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/statistics.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/rpath.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    copying src/rdiff_backup/compare.py -> build/lib.linux-x86_64-3.9/rdiff_backup
    running build_ext
    building 'rdiff_backup.C' extension
    creating build/temp.linux-x86_64-3.9
    creating build/temp.linux-x86_64-3.9/src
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/python3.9 -c src/cmodule.c -o build/temp.linux-x86_64-3.9/src/cmodule.o
    gcc -pthread -shared build/temp.linux-x86_64-3.9/src/cmodule.o -L/usr/lib64 -o build/lib.linux-x86_64-3.9/rdiff_backup/C.cpython-39-x86_64-linux-gnu.so
    building 'rdiff_backup._librsync' extension
    gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -I/usr/include/python3.9 -c src/_librsyncmodule.c -o build/temp.linux-x86_64-3.9/src/_librsyncmodule.o
    src/_librsyncmodule.c:25:10: fatal error: librsync.h: No such file or directory
       25 | #include <librsync.h>
          |          ^~~~~~~~~~~~
    compilation terminated.
    error: command '/usr/bin/gcc' failed with exit code 1
    ----------------------------------------
ERROR: Command errored out with exit status 1: /usr/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-czl32xcz/rdiff-backup/setup.py'"'"'; __file__='"'"'/tmp/pip-install-czl32xcz/rdiff-backup/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-_znlo7sn/install-record.txt --single-version-externally-managed --compile --install-headers /usr/include/python3.9/rdiff-backup Check the logs for full command output.
```

## More information

If you can, please repeat the action leading to the error, adding the option `-v9` and attach the output to this bug report.

--- a/README.md
+++ b/README.md
@@ -117,14 +117,12 @@ sudo dnf install python3-pip python3-setuptools py3libacl python3-pyxattr
 sudo pip3 install rdiff-backup
 ```
 
-> **NOTE:** If your platform is not i386 or amd64, e.g. ARM/MIPS/PowerPC/etc,
-  you may need the build dependencies `python3-devel`, `librsync-dev`.
-
-### Other Linux (From PyPi)
+### Other Linux and UN\*X-oid systems, e.g. BSD (From PyPi)
 
 You need to make sure that the following requirements are met:
 
 * Python 3.6 or higher
+* pip3 e.g. installed with `curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py; sudo python3 get-pip.py`.
 * librsync 1.0.0 or higher
 * pylibacl (optional, to support ACLs)
 * pyxattr (optional, to support extended attributes) - the xattr library (without py) isn't regularly tested but should work and you will be helped
@@ -132,11 +130,15 @@ You need to make sure that the following requirements are met:
   (or alternatively setuptools)
 * SSH for remote operations
 
+Then you should only need to call the following before you can use rdiff-backup:
+
 ```
-curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
-sudo python3 get-pip.py
 sudo pip3 install rdiff-backup
 ```
+
+> **NOTE:** especially if your platform is not i386 or amd64, e.g. ARM/MIPS/PowerPC/etc,
+  but also if the pip3 installation fails with `include [...].h` files missing,
+  you may need the build dependencies named like `python3-devel` or `librsync-dev`.
 
 ### Windows
 

